README.md

```diff
@@ -1 +1,90 @@
-# breakers
\ No newline at end of file
+# Breakers
+
+Breakers is a Ruby gem that implements the circuit breaker pattern for Ruby using a Faraday middleware. It is designed to handle the case
+where your app communicates with one or more backend services over HTTP and those services could possibly go down. Data about the success
+and failure of requests is recorded in Redis, and the gem uses this to determine when an outage occurs. While a service is marked as down,
+requests will continue to flow through every minute to check if it has returned to being alive.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'breakers'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install breakers
+
+## Usage
+
+### Getting Connected
+
+The gem allows you to define your services like this:
+
+```ruby
+service = Breakers::Service.new(name: 'messaging') do |request_env|
+  request_env.uri.host =~ /.*messaging\.va\.gov/
+end
```

Comment:
Why would the gem not depend on `faraday-middleware` gem so that it can be added to an existing client specific connection? see: [sample faraday-middleware](https://github.com/lostisland/faraday_middleware/blob/master/lib/faraday_middleware/instrumentation.rb)